removal of the Bio.Alphabet - it is not part of Biopython anymore (since v1.78)
see also https://biopython.org/wiki/Alphabet
author: Pavel Tomanek (INUITS)
--- bin/ncbi/phyluce_ncbi_prep_uce_align_files_for_ncbi.orig	2024-02-01 15:53:37.732931000 +0100
+++ bin/ncbi/phyluce_ncbi_prep_uce_align_files_for_ncbi	2024-02-01 16:16:42.885167000 +0100
@@ -18,7 +18,6 @@
 from Bio import AlignIO
 from Bio.Seq import Seq
 from Bio.SeqRecord import SeqRecord
-from Bio.Alphabet import IUPAC
 
 from phyluce import ncbi
 from phyluce.log import setup_logging
@@ -127,7 +126,7 @@
                             .upper()
                         )
                         new_record = SeqRecord(
-                            Seq(new_seq, IUPAC.IUPACAmbiguousDNA()),
+                            Seq(new_seq),
                             id=new_id,
                             name="",
                             description="",
